docs/guide/terraform-guide.md

```diff
@@ -0,0 +1,181 @@
+title: Sematext Terraform Guide
```

Comment:
@adnanrahic do you think this should be under Guides?  Or should be it under Integrations?

Reply:
We plan on removing guides entirely down the road. Let's have it under Integrations instead.

Reply:
Moved to new Orchestration section as discussed (Terraform orchestrates - it's not an Integration).